matrimony_admin/views.py

```
@@ -2,12 +2,12 @@
 import os
 from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render, redirect
-from django.views.generic import TemplateView,DetailView,ListView
+from django.views.generic import TemplateView, DetailView, ListView
```

Ruff

`django.views.generic.DetailView` imported but unused

Remove unused import: `django.views.generic.DetailView`

(F401)